### DISCUSSION TITLE
esptool: error: unrecognized arguments

### DISCUSSION THREAD

**Discussion by bantle1984** [emporia-vue-local/esphome#301] [Q&A]
this is the error i get from 

python3 -m esptool run /Users/brad/Desktop/project.yaml

esptool: error: unrecognized arguments: /Users/brad/Desktop/project.yamlpython3 -m esptool upload /Users/brad/Desktop/project.yaml

is there a issue with my command line or could the yaml file have issues?

**Comment by flaviut** [comment]
Config looks probably fine but you're getting your commands mixed up. `esphome` and `esptool` are different unrelated commands

**Comment by bantle1984** [comment]
brad@Brads-Mac-mini ~ % python3 -m esphome run /Users/brad/Desktop/project.yaml


/Library/Developer/CommandLineTools/usr/bin/python3: No module named esphome

seems like there is no esphome installed?

**Comment by bantle1984** [comment]
sorry got esphome installed now but seems some issues with config file

while parsing a flow mapping
  in "/Users/brad/Desktop/project.yaml", line 1, column 1
expected ',' or '}', but got '{'
  in "/Users/brad/Desktop/project.yaml", line 2, column 34
